magda-web-client/src/helpers/promiseAny.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Take an array of promises and return a new promise that will be resolved
+ * when any of promises is resolved and will be rejected when all promised are rejected.
+ *
+ * @export
+ * @param {Promise<any>[]} items
+ * @returns {Promise<any>}
+ */
+export default function promiseAny(items: Promise<any>[]): Promise<any> {
```

Comment:
Not crazy about this name... how do you feel about something like `raceSuccessOnly`, seeing how it shares some of its semantics with `promise.race` ?

Reply:
I actually try to replicate the functionality of this coming promise API (Promise.any):
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any
but no preference on any particular names --- let me know if you prefer a `raceSuccessOnly` or we just leave as it is

Reply:
Ooh ok fair enough. Maybe mention that in the jsdoc then?

Reply:
👍 That's a good point~ Have added more comments in the jsdoc

Reply:
:+1: I'll put that link the javadoc in my spelling changes commit :)